ext/tls/tls_key.rs

```diff
@@ -1,15 +1,24 @@
+use crate::load_certs;
```

Comment:
Drive by, why is copyright checker not complaining about this file?

Reply:
I wonder if it's because there's a conflict. The CI seems oddly wedged on this PR.